sdk/dotnet/Pulumi.Automation/PreviewOptions.cs

```diff
@@ -20,6 +20,8 @@ public sealed class PreviewOptions : UpdateOptions
 
         public PulumiFn? Program { get; set; }
 
+        public string? Plan {get; set; }
```

Comment:
Since this is public facing, it really should have a comment explaining what it does. I had to remember how the CLI worked before I figured out that this represents the path where the plan is stored, not the plan itself.